<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>iphone_calculator</class>
 <widget class="QFrame" name="iphone_calculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>331</width>
    <height>534</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>331</width>
    <height>534</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>331</width>
    <height>534</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>iPhone Calculator</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QFrame{
	background-color:rgb(0, 0, 0)
}

QPushButton{
	background-color:rgb(51, 51, 51);
	border:none;
	border-radius:35px;
	geometry:71 x 71;
	color:rgb(255, 255, 255);
	font: 26pt &quot;SF Pro Text&quot;;
}

QPushButton:pressed{
	background-color:rgb(115, 115, 115)
}</string>
  </property>
  <widget class="QPushButton" name="butClear">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:rgb(167, 166, 167);
	color:rgb(0, 0, 0);
	font: 500 20pt &quot;SF Pro Text&quot;;
}
QPushButton:pressed{background-color:rgb(216, 216, 216)}</string>
   </property>
   <property name="text">
    <string>C</string>
   </property>
  </widget>
  <widget class="QPushButton" name="butPM">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>130</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:rgb(167, 166, 167);
	color:rgb(0, 0, 0);
	font: 500 16pt &quot;SF Pro Text&quot;;
}
QPushButton:pressed{background-color:rgb(216, 216, 216)}</string>
   </property>
   <property name="text">
    <string>+/-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="butPro">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>130</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:rgb(167, 166, 167);
	color:rgb(0, 0, 0);
	font: 500 20pt &quot;SF Pro Text&quot;;
}
QPushButton:pressed{background-color:rgb(216, 216, 216)}</string>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QPushButton" name="butDiv">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>130</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	Background-color:rgb(255, 161, 4);
}
QPushButton:pressed{
	background-color:rgb(251, 199, 141)
}</string>
   </property>
   <property name="text">
    <string>รท</string>
   </property>
  </widget>
  <widget class="QPushButton" name="butMulti">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>210</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	Background-color:rgb(255, 161, 4);
}
QPushButton:pressed{
	background-color:rgb(251, 199, 141)
}</string>
   </property>
   <property name="text">
    <string>ร</string>
   </property>
  </widget>
  <widget class="QPushButton" name="butMin">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>290</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	Background-color:rgb(255, 161, 4);
}
QPushButton:pressed{
	background-color:rgb(251, 199, 141)
}</string>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="butAdd">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>370</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	Background-color:rgb(255, 161, 4);
}
QPushButton:pressed{
	background-color:rgb(251, 199, 141)
}</string>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="butEqu">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>450</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	Background-color:rgb(255, 161, 4);
}
QPushButton:pressed{
	background-color:rgb(251, 199, 141)
}</string>
   </property>
   <property name="text">
    <string>=</string>
   </property>
  </widget>
  <widget class="QPushButton" name="but7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="but8">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>210</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="but9">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>210</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="but4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="but5">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>290</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="but6">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>290</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="but1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="but2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>370</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="but3">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>370</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="but0">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>450</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SF Pro Text</family>
     <pointsize>26</pointsize>
     <italic>false</italic>
     <bold>false</bold>
     <stylestrategy>PreferDefault</stylestrategy>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="butZap">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>450</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>,</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>291</width>
     <height>91</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 38pt &quot;SF Pro Text&quot;;
color:rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
